addon/components/lt-body.js

```diff
@@ -239,6 +290,61 @@ export default Component.extend({
     this.set('useVirtualScrollbar', fixedHeader || fixedFooter);
   },
 
+  didUpdateAttrs({ newAttrs, oldAttrs }) {
+    this._super(...arguments);
+
+    let newScrollTo = get(newAttrs, 'scrollTo.value');
+    let oldScrollTo = get(oldAttrs, 'scrollTo.value');
+    let newScrollToRow = get(newAttrs, 'scrollToRow.value');
+    let oldScrollToRow = get(oldAttrs, 'scrollToRow.value');
+
+    let targetScrollOffset = 0;
+    let hasReachedTargetScrollOffset = true;
+
+    if (oldScrollTo !== newScrollTo) {
+      targetScrollOffset = newScrollTo;
+      if (newScrollTo > 0) {
+        hasReachedTargetScrollOffset = false;
+      }
+    } else if (oldScrollToRow !== newScrollToRow) {
```

Comment:
Can this just be `else if (newScrollToRow instanceof Row && oldScrollToRow !== newScrollToRow)`?